torch_xla/experimental/custom_kernel.py

```diff
@@ -490,11 +490,8 @@ def non_xla_attetion(q, k, v, attention_type):
         f'XLA {attention_type} attention should only be applied to tensors on XLA device'
     )
 
-  # perform a regular attention if input tensors are not on XLA device.
-  attn_weight = q @ k.transpose(-2, -1)
-  attn_weight = torch.nn.functional.softmax(attn_weight, dim=-1)
-  attn_output = attn_weight @ v
-  return attn_output
+  # Return orignal shape of q.
+  return torch.empty_like(q)
```

Comment:
Do you know if this actually initialize anything? I hope not.

Reply:
yea it is worth checking out, in above we have a warning about the `q` should be on meta device, I think running ops on meta_tensor will not allocate any device memory.

Reply:
According to https://pytorch.org/docs/stable/generated/torch.empty_like.html, it seems like emtpy_like returns an uninitialized tensor. Along with the meta tensor check above, I think we should be good.